.gitignore

```diff
@@ -5,3 +5,4 @@ lib/
 msg_gen/
 srv_gen/
 devel/
+.vscode
```

Comment:
@Dillon-Roller added this to avoid committing your project files from your editor.